src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -306,9 +311,10 @@ export const TradeInput = memo(() => {
           </Button>
         </Tooltip>
         {hasUserEnteredAmount &&
-          (!applyThorSwapAffiliateFees || activeSwapperName !== SwapperName.Thorchain) && (
-            <DonationCheckbox isLoading={isLoading} />
-          )}
+          (!applyThorSwapAffiliateFees ||
+            ![SwapperName.Thorchain, SwapperName.CowSwap].includes(
```

Comment:
This would break the `REACT_APP_FEATURE_THOR_SWAP_AFFILIATE_FEES` feature flag.

I.e. if the flag is off we'll _always_ show the donation checkbox, _including_ for CoW (which we should never show).

Reply:
🧠 fixing ser